### ISSUE TITLE
Sato-Tate groups for Dirichlet L-functions

### ISSUE THREAD

**Issue by davidfarmer** [LMFDB/lmfdb#1184]
The Sato-Tate groups of Dirichlet L-functions are listed in the form "mu(N)",
which is supposed to represent the subgroup the unit circle consisting of the Nth roots of 1.

These should be replaced by an official name, and linked to the newly created Sato-Tate material.

**Comment by JohnCremona** [comment]
Agreed.  Drew?  But is it only the L-function which has a ST group, or does the character itself?
The same applied for elliptic curves/Q: the ST group is shown on the L-function page but not as a link to the ST group page.
For ec/nf we now show ST groups but not yet for their L-functions.

**Comment by AndrewVSutherland** [comment]
ST group links now appear on Dirichlet L-function pages and as friends of Dirichlet characters (PR #1658 and PR #1703).
